### ISSUE TITLE
Mask Probabilty for Task-specific Pre-training

### ISSUE THREAD

**Issue by Muennighoff** [uclanlp/visualbert#13]
Hi, in your paper you mention that task-specific pre-training is also using masked-language modelling similar to task-agnostic pre-training. However, I cannot find any mask probability for the task-specific pre-training .json files. Why is no probability specified & did you use the same 15% probability as for task agnostic pre-training?

Sorry perhaps I'm missing something here -- Thanks for the help!

**Comment by liunian-harold-li** [comment]
Sorry I did not make it clear in the paper. I still use the 15% probability
and hard code it in the code (since I did not plan to tune this
probability).

**Comment by Muennighoff** [comment]
Great thanks, closing the issue!
